horovod/common/operations.cc

```diff
@@ -1772,5 +1773,39 @@ Status EnqueueTensorBroadcast(std::shared_ptr<OpContext> context,
   return Status::OK();
 }
 
+// MPI must be initialized and the background thread must be running before
+// this function is called.
+Status EnqueueJoin(std::shared_ptr<OpContext> context,
+                              std::shared_ptr<ReadyEvent> ready_event,
```

Comment:
Nit: use `clang-format` to format code w/o unaligned spaces

Reply:
Done.